config/config.yaml

```diff
@@ -18,4 +18,7 @@ dependencies:
         activate_endpoint: "/api/account/activate"
         client_id: "ons@ons.gov"
         client_secret: "password"
-
+    dummy-service:
```

Comment:
Yes good point. In fact I'll do that now as we can then deploy this version and have it do something potentially useful, or at least testable.